### PR TITLE
Improve/parachain workflow

### PR DESCRIPTION
**Pull request by GopherJ**
close: #266 
close: #272

### CODE REVIEW

File: Dockerfile.vanilla [hunk 1]
```diff
@@ -0,0 +1,30 @@
+FROM paritytech/ci-linux:production as builder
```
Comment (MrPai) [inline Dockerfile.vanilla:1]: Can we merge these two Dockerfiles into one?

**Review by MrPai — COMMENTED** [review]
`root_key` for  heiko-dev chain spec should be common account, such as Alice;
and there is no need to include "Parallel team accounts" in heiko-dev

File: Dockerfile.vanilla (hunk 1 above)
Comment (GopherJ) [reply Dockerfile.vanilla:1]: the default command isn't the same

**Comment by GopherJ** [comment]
the others are ok now

File: Makefile [hunk 2]
```diff
@@ -79,14 +79,19 @@ resources:
 	target/release/parallel export-genesis-state --parachain-id 200 > ./resources/para-200-genesis
 	target/release/parallel export-genesis-wasm > ./resources/para-200.wasm
 
-.PHONY: register-parachain
-register-parachain:
-	polkadot-js-api --seed "//Alice" --sudo tx.parasSudoWrapper.sudoScheduleParaInitialize 200 "{ \"genesisHead\":\"$(shell cat ./resources/para-200-genesis)\", \"validationCode\":\"$(shell cat ./resources/para-200.wasm)\", \"parachain\": true }"
+.PHONY: docker-resources
+docker-resources:
+	docker run --rm  parallelfinance/parallel:latest parallel export-genesis-state --chain heiko-dev --parachain-id 200 > ./resources/para-200-genesis
```
Comment (MrPai) [inline Makefile:84]: parachain-id should start from 2000, you can refer to https://substrate.dev/cumulus-workshop/#/en/2-relay-chain/2-reserve?id=reserve-a-paraid

Comment (GopherJ) [reply Makefile:84]: we haven't decided our paraid yet so 200 is only for testing

**Comment by GopherJ** [comment]
ok now it starts to produce blocks

**Review by MrPai — APPROVED** [review]
looks good to me

**Pull request merged**